charts/internal/gvisor/templates/rbac.yaml

```
@@ -0,0 +1,57 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: gvisor
+  namespace: kube-system
+spec:
+  privileged: true
+  volumes:
+    - 'configMap'
+    - 'secret'
+    - 'hostPath'
+  hostNetwork: false
+  hostIPC: true
+  hostPID: true
+  allowedHostPaths:
+    - pathPrefix: /
+  runAsUser:
+    rule: 'RunAsAny'
+  seLinux:
+    rule: 'RunAsAny'
+  supplementalGroups:
+    rule: 'RunAsAny'
+  fsGroup:
+    rule: 'RunAsAny'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gvisor
+rules:
+  - apiGroups:
+      - policy
```

```suggestion
      - extensions
      - policy
```

PSP in `extensions` API group is deprecated, but just to be sure for older Kubernetes versions.